code/datums/components/suicide_count.dm

```diff
@@ -0,0 +1,60 @@
+// no extra data displayed (exclusively using callback)
+#define SUICIDE_VIS_NONE 0
+// displays suicide data to holyroles (chaplain)
+#define SUICIDE_VIS_HOLY 1
+// displays suicide data to everyone
+#define SUICIDE_VIS_ALL 2
```

Comment:
These need to be in a new defines file in `code/__DEFINES/suicide.dm`. Otherwise, we can't guarantee a file will have it unless it so happens to be after datums/components.